src/cmd/vm.rs

```diff
@@ -20,136 +27,231 @@ pub struct Args {
-    #[argh(option, short = 'b')]
+    #[argh(option)]
     board: String,
 
-    /// reuse the VM image. It is true by default.
+    /// reuse the VM image. It is true by default. If you want to disable it,
+    /// use `--reuse-disk-image flase`.
     #[argh(option, default = "true")]
     reuse_disk_image: bool,
 
-    /// specifies the display device for the VM. "vnc" (default) creates a VNC
-    /// display on localhost:5900; "virgl" enables guest 3D acceleration,
-    /// assuming X11 (or use xvfb-run) and qemu compiled with GTK/virgl support;
-    /// "none" removes the VGA device, which can be used to simulate the GCE
-    /// environment (i.e. L1 betty).
-    #[argh(option)]
-    display: Option<String>,
+    /// start betty with rootfs verification
+    #[argh(switch)]
+    rootfs_verify: bool,
 
     /// the ChromeOS version to use (e.g. R72-11268.0.0). Alternatively,
     /// postsubmit builds since R96-14175.0.0-53101 can also be specified. It is
     /// the latest version by default.
     #[argh(option)]
     version: Option<String>,
 
-    /// the android version to push. This is passed to push_to_device.py. e.g.
-    /// cheets_x86/userdebug/123456
-    #[argh(option, short = 'a')]
-    android_build: Option<String>,
+    /// path to betty VM image to start. It has priority over --board and
+    /// --version (they will be ignored)
+    #[argh(option)]
+    vm_image: Option<String>,
 
-    /// extra arguments
+    /// extra arguments to pass to betty.sh. You can pass other options like
+    /// --extra-args "options".
     #[argh(option)]
-    extra_args: Vec<String>,
+    extra_args: Option<String>,
 }
 
 fn run_start(args: &ArgsStart) -> Result<()> {
-    let mut vec = Vec::new();
-    vec.append(&mut vec!["--board", &args.board]);
+    let dir = find_betty_script(&args.arc)?;
+
+    let mut options = Vec::new();
+    options.append(&mut vec!["--board", &args.board]);
     if !args.reuse_disk_image {
-        vec.append(&mut vec!["--reset_image"])
+        options.append(&mut vec!["--reset_image"]);
     }
-    if let Some(display) = &args.display {
-        vec.append(&mut vec!["--display", display]);
+    if args.rootfs_verify {
+        options.append(&mut vec!["--nodisable_rootfs"]);
     }
     if let Some(version) = &args.version {
-        vec.append(&mut vec!["--release", version]);
+        options.append(&mut vec!["--release", version]);
+    }
+    if let Some(vm_image) = &args.vm_image {
+        options.append(&mut vec!["--vm_image", vm_image]);
     }
-    if let Some(android_build) = &args.android_build {
-        vec.append(&mut vec!["--android_build", android_build]);
+    options.append(&mut vec!["--display", "none"]);
+    if let Some(extra_args) = &args.extra_args {
+        options.append(&mut vec![extra_args]);
     }
 
-    let mut options: Vec<String> = vec.iter().map(|s| s.to_string()).collect();
+    let arg = options.join(" ");
+    run_betty(&dir, "start", &arg)?;
```

Comment:
maybe arg should be passed as a slice of strings and joined inside the `run_betty` function